packages/neos-ui/src/Containers/SecondaryToolbar/LoadingIndicator/style.css

```diff
@@ -1,10 +1,9 @@
 .loadingIndicator__container {
-    left: 50%;
-    top: 39px;
+    left: 0;
+    top: 0;
     position: absolute;
-    transform: translateX(-50%);
-    padding: 0 320px;
-    width: 100%;
+    width: 100vw;
+    z-index: 2;
```

Comment:
Hey, we have a clever system of managing z-index, put it there: https://github.com/neos/neos-ui/blob/master/packages/build-essentials/src/styles/styleConstants.js#L10